src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.html

```diff
@@ -1,31 +1,64 @@
-              placeholder="Description goes here"
-              class="space-description form-control"
-              *ngIf="isEditing && (isEditable() | async)"
-              (keyup.enter)="onUpdateDescription(description.value)"
-              (keydown.enter)="preventDef($event)"
-              #description>{{space?.attributes.description}}</textarea>
-    <div *ngIf="isEditing">
-      <button id="_btn_cancel"
-              class="pull-right btn btn-default edit-space-description-btn"
-              (click)="stopEditingDescription()">
-        <span class="fa fa-close"></span>
-      </button>
-      <!--Save icon-->
-      <button id="workItemTitle_btn_save"
-              class="pull-right btn btn-primary edit-space-description-btn "
-              (click)="saveDescription()" >
-        <span class="fa fa-check"></span>
-      </button>
+<f8-feature-toggle featureName="newSpaceDashboard" restricted="dev">
+  <div class="edit-space-description-widget" user-level>
+    <div class="col-sm-3 f8-space-masthead-space-info">
+      <div>
+        <span class="f8-space-masthead-name">{{space?.attributes.name | spaceName}}</span> <span class="f8-space-masthead-owner">bobross_925</span>
+      </div>
+      <div>
+        <span class="pficon pficon-users fa-lg"></span> 59
+      </div>
+    </div>
+    <div class="f8-space-masthead-vertical-divider"></div>
+    <div class="col-sm-9 f8-space-masthead-description">
+      <!-- There must be no spaces here, otherwise we end up with random whitespace in the editable-->
+      <div class="form-group">
+        <div class="f8-space-masthead-description-title"></div>
+        <textarea row="3"
+                  placeholder="Description goes here"
+                  class="space-description form-control"
+                  *ngIf="isEditing && (isEditable() | async)"
+                  (keyup.enter)="onUpdateDescription(description.value)"
+                  (keydown.enter)="preventDef($event)"
+                  #description>{{space?.attributes.description}}</textarea>
+        <div class="f8-space-masthead-description-edit-buttons" *ngIf="isEditing">
+          <button id="_btn_cancel_description"
+                  class="pull-right btn btn-default edit-space-description-btn"
+                  (click)="stopEditingDescription()">
+            Cancel
+          </button>
+          <!--Save icon-->
+          <button id="workItemTitle_btn_save_description"
+                  class="pull-right btn btn-primary edit-space-description-btn "
+                  (click)="saveDescription()" >
+            Save
+          </button>
+        </div>
+        <p *ngIf="!isEditing || !(isEditable() | async)" (click)="startEditingDescription()" class="f8-space-masthead-description-body edit-space-description-pencil truncate-multiline">
+          {{space?.attributes.description || "Enter a description"}}
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="old-edit-space-description-widget" default-level>
```

Comment:
another `default-level`

Reply:
`default-level` and `user-level` needs to be applied on every component that needs to match the "local development only" criteria. This will be done on many components for the Dashboard. Corinne and Xavier are aware, as they suggested this usage for the time being.